src/utils/MemoryFS.js

```
@@ -0,0 +1,471 @@
+      const err = new Error(`ENOTDIR: not a directory, open '${p}'`)
+      err.code = 'ENOTDIR'
+      throw err
+    }
+
+    let content
+    if (typeof data === 'string') {
+      const encoding =
+        typeof options === 'string'
+          ? options
+          : options && options.encoding
+          ? options.encoding
```

ESLint

[error] Insert `··`

(prettier/prettier)